wikidataQuery.py

```diff
@@ -3,13 +3,17 @@
 import settings
 import traceback
 import json
+import simplejson
```

Comment:
Please make clear that this also needs to be installed. Also, could you add both this and TextBlob to the README?

Reply:
simplejson does not need to be installed though